lib/models/user-identity.js

```diff
@@ -156,7 +156,16 @@ function UserIdentity(UserIdentity) {
             },
             function(err, i) {
               // Find the user for the given identity
-              return identity.user(function(err, user) {
+              // When using custom user models, the relation between Identity and User does not seem to work.
+              // Fetching the user model out of the identity.
+              if (err) {
+                cb(err, identity);
```

Comment:
Should this `return` here? If the fall-through behaviour is expected then there should probably be a comment indicating why.